src/Configurator/DockerComposeConfigurator.php

```diff
@@ -43,16 +43,20 @@ public function configure(Recipe $recipe, $config, Lock $lock, array $options =
 
         $rootDir = $this->options->get('root-dir');
         foreach ($this->normalizeConfig($config) as $file => $extra) {
-            if (
-                (null === $dockerComposeFile = $this->findDockerComposeFile($rootDir, $file)) ||
-                $this->isFileMarked($recipe, $dockerComposeFile)
-            ) {
+            $dockerComposeFile = $this->findDockerComposeFile($rootDir, $file);
+            if (null === $dockerComposeFile) {
+                $dockerComposeFileName = preg_replace('/\.yml$/', '.yaml', $file);
+                $dockerComposeFile = $rootDir.'/'.$dockerComposeFileName;
+                file_put_contents($dockerComposeFile, "version: '3'\n");
+                $this->write(sprintf('  Created <fg=green>"%s"</>', $dockerComposeFileName));
+            }
+            if ($this->isFileMarked($recipe, $dockerComposeFile)) {
                 continue;
             }
 
             $this->write(sprintf('Adding Docker Compose definitions to "%s"', $dockerComposeFile));
 
-            $offset = 8;
+            $offset = 2;
```

Comment:
The default indentation for Docker Compose file is 2 spaces. This was not caught before as tests always used an existing file.